### PR TITLE
add v.1.4.0 release

### PR DESCRIPTION
**Pull request by googol88**

### CODE REVIEW

File: data/releases.yaml [hunk 1]
```diff
@@ -1,3 +1,10 @@
+- version: v1.4.0
+  changes:
+  - add option to block exact website URL
+  - treat subdomains as separate websites when blocking sites
```
Comment (jackyzha0) [inline data/releases.yaml:4]: lets add an example here

Comment (googol88) [reply data/releases.yaml:4]: agreed, I'll take the one I added in the PR 👍

Comment (googol88) [reply data/releases.yaml:4]: agreed, I'll use the examples from the PRs I made 👍

Comment (googol88) [reply data/releases.yaml:4]: p.s. are we meeting today?

**Review by jackyzha0 — APPROVED** [review]

**Pull request merged**